### PR TITLE
Update setup commands

### PR DESCRIPTION
**Pull request by Coder-DG**
- Clarifies the MTU size setup
- Changes the setup commands to work without changing anything

### CODE REVIEW

File: cloudify_docker_image/README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@
     ```
     sudo yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
     sudo yum install -y docker-ce
-    sudo usermod -aG docker your_username
+    sudo usermod -aG docker $(whoami)
```
Comment (geokala) [inline cloudify_docker_image/README.md]: You could make this ${USER} on any systems we support for this without breaking anything I think- that'd avoid calling out to an extra command. (I mean, in this case it's not saving any effort, but as a general rule it's probably better)

Comment (Coder-DG) [reply cloudify_docker_image/README.md]: Changed

File: cloudify_docker_image/README.md [hunk 2]
```diff
@@ -28,7 +17,7 @@ network adapter's mtu size and edit the docker service file with
 ```
 # The path also might be at /usr/lib/systemd/system/docker.service
 sudo vi /etc/systemd/system/docker.service
-# At 'ExecStart=' add '--mtu HOST_MTU_SIZE' flag to the end of the line
+# At 'ExecStart=' add '--mtu <HOST_MTU_SIZE>' flag to the end of the line
```
Comment (geokala) [inline cloudify_docker_image/README.md]: Can you do this with sed?

Comment (Coder-DG) [reply cloudify_docker_image/README.md]: I could but I don't think it's worth the time (gotta get the correct interface mtu and then replace it in the file)

File: cloudify_docker_image/centos_mtu_fix.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
+echo "Enter docker.service file path:"
```
Comment (geokala) [inline cloudify_docker_image/centos_mtu_fix.sh]: Why doesn't this have a hashbang?

Also, please rename all of these scripts to not have the .sh- the language used within them is an implementation detail, no need to make it part of the interface (this also means that they can be re-engineered in python or anything else if we ever want to).

**Review by geokala — CHANGES_REQUESTED** [review]

File: cloudify_docker_image/centos_mtu_fix.sh (hunk 3 above)
Comment (Coder-DG) [reply cloudify_docker_image/centos_mtu_fix.sh]: Just forgot to put one, now it does.

**Review by geokala — APPROVED** [review]

**Pull request merged**